.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "npm" # npm as the package ecosystem
+    directory: "/" # root directory where package.json is located
```

Comment:
@davidwiese I think this might be `products/statement-generator/` because the `.github` directory is located in the root of the repo, but _the app itself_ is in the one stated above. That folder is where the package.json is located and what you likely move into before starting the app locally.

Reply:
Good catch, updated!